### PR TITLE
Fix(frontend): adjust sidebar label 529

### PR DESCRIPTION
**Pull request by Torresmorah**
### What does this PR do?

- resolves [#529](https://github.com/eoscostarica/guide.eoscostarica.io/issues/529)
- fix the wrong language in "configuración de nodeos" page

### Steps to test
1. Check that the changed sidebar labels are correct
1. Check the translation in the `opciones-configuracion-nodeos.md` file

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in Spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/herramientas/ejemplos-configuracion-nodeos.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: ejemplos-configuracion-nodeos
 title: Ejemplos para la Configuración de Nodeos 
-sidebar_label: Ejemplos de Configuración de Nodeos 
+sidebar_label: Colección de Configuración Nodeos
```
Comment (xavier506) [inline docs/herramientas/ejemplos-configuracion-nodeos.md:4]: Pongamos esto en plural

`Colección de Configuraciones Nodeos`

File: docs/herramientas/ejemplos-contratos-inteligentes-eosio.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 id: ejemplos-contratos-inteligentes-eosio
 title: Ejemplos de Contratos Inteligentes EOSIO
-sidebar_label: Ejemplos de Contratos Inteligentes EOSIO
+sidebar_label: Colección de Contratos Inteligentes
```
Comment (xavier506) [inline docs/herramientas/ejemplos-contratos-inteligentes-eosio.md:4]: Que te parece si le ponemos: `Algunos Contratos Inteligentes EOSIO`?

File: docs/herramientas/ejemplos-configuracion-nodeos.md (hunk 1 above)
Comment (Torresmorah) [reply docs/herramientas/ejemplos-configuracion-nodeos.md:4]: Queda en dos líneas

File: docs/herramientas/ejemplos-contratos-inteligentes-eosio.md (hunk 2 above)
Comment (xavier506) [reply docs/herramientas/ejemplos-contratos-inteligentes-eosio.md:4]: `Ejemplos de Contratos Inteligentes` ?

File: docs/herramientas/ejemplos-configuracion-nodeos.md (hunk 1 above)
Comment (xavier506) [reply docs/herramientas/ejemplos-configuracion-nodeos.md:4]: OK y `Ejemplos de Configuración Nodeos` ?

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**